src/blocks/scratch3_looks.js

```diff
@@ -344,7 +344,13 @@ class Scratch3LooksBlocks {
                        requestedCostume === 'next backdrop') {
                 target.setCostume(target.currentCostume + 1);
             } else if (requestedCostume === 'random backdrop') {
-                target.setCostume(Math.floor(Math.random() * target.getCostumes().length));
+                if ((numCostumes = target.getCostumes().length) > 1) {
```

Comment:
Huh? You're reviewing an outdated version - that's exactly what I did.

Reply:
Whoops. I'm not sure why GitHub decided to notify me about a commit here, but then show me the changes of the oldest new commit instead of all the new changes.